### ISSUE TITLE
softmax is applied to webgl, but not webassembly or webgpu

### ISSUE THREAD

**Issue by kylemcdonald** [mil-tokyo/webdnn#591]
Based on my testing, I found that the final softmax layer of a network is being used with the webassembly and webgpu backend, but not with the webgl backend.

I tested on a small ResNet architecture and a VGG16 architecture, both built in Keras.

If this is unclear, let me know and I can provide more details!

**Comment by Kiikurage** [comment]
Sorry for confusing you, but currently WebGL backend doesn't support softmax operation. In case that softmax is used only in the last of your model, conversion is successed (but no softmax is included in the converted model and you need to apply it by JS).

Also, please see [this comment](https://github.com/mil-tokyo/webdnn/issues/588#issuecomment-333807690).

**Comment by kylemcdonald** [comment]
Ah... your comment in #561, I didn't read it closely enough.

It would be very useful to output a warning during conversion. Otherwise more people might think "webgl is broken".

**Comment by Kiikurage** [comment]
FYI: In latest revision, I implement SoftMax in WebGL backend.
